### PR TITLE
Contributing md file

### PR DESCRIPTION
**Pull request by darrenvechain**

### CODE REVIEW

**Comment by claytonneal** [comment]
Just wondering if we can work a link to the VIP repo from the contributing doc file! thanks

**Comment by darrenvechain** [comment]
Yepp, will 

> Just wondering if we can work a link to the VIP repo from the contributing doc file! thanks

File: README.md [hunk 1]
```diff
@@ -151,54 +155,33 @@ Release [v2.0.4](https://github.com/vechain/thor/releases/tag/v2.0.4) changed th
...
 
-* [faucet.vecha.in](https://faucet.vecha.in) by *VeChain Foundation*
+* [faucet.vecha.in](https://faucet.vecha.in) by *Vechain Foundation*
 
 ## API
 
 Once `thor` has started, the online *OpenAPI* doc can be accessed in your browser. e.g. [http://localhost:8669/](http://localhost:8669) by default.
 
-[![Thorest](https://raw.githubusercontent.com/vechain/thor/master/thorest.png)](http://localhost:8669/)
+[![Thorest](./thorest.png)](http://localhost:8669/)
```
Comment (libotony) [inline README.md:170]: How about changing this in pr #628

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
+
+# Contributor Covenant Code of Conduct
```
Comment (libotony) [inline CODE_OF_CONDUCT.md:2]: Is this a full copy from https://www.contributor-covenant.org/version/2/1/code_of_conduct/code_of_conduct.md? If so I'll skip readding it :-)

File: README.md (hunk 1 above)
Comment (darrenvechain) [reply README.md:170]: Yes, it's a copy and paste minus the support email

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (darrenvechain) [reply CODE_OF_CONDUCT.md:2]: Yes, its a copy and paste minus the support email

**Comment by darrenvechain** [comment]
@claytonneal I mentioned the VIPs in the contributing doc.

@libotony I made the whitespace change

File: README.md [hunk 3]
```diff
@@ -151,54 +155,33 @@ Release [v2.0.4](https://github.com/vechain/thor/releases/tag/v2.0.4) changed th
...
 
 ## Acknowledgement
 
 A special shout out to following projects:
 
 * [Ethereum](https://github.com/ethereum)
-* [Swagger](https://github.com/swagger-api)
+* [Stoplight Elements](https://github.com/stoplightio/elements)
```
Comment (libotony) [inline README.md:177]: Do not forget this

File: README.md (hunk 1 above)
Comment (libotony) [inline README.md:170]: Keep the full URL, it's for other sources displaying the image, for example https://hub.docker.com/r/vechain/thor

**Comment by darrenvechain** [comment]
@libotony I resolved the latest

**Review by libotony — APPROVED** [review]

**Pull request merged**